6/6.css

```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+.blacksmile {
+  width: 200px;
+  height: 200px;
+  margin-bottom: 30px;
+}
+.fa-smile {
+  color: #000000;
+  font-size: 200px;
+}
+.ytbutton {
+  width: 200px;
+  height: 200px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #d7d7d7;
+  margin-bottom: 30px;
+}
+.fa-youtube {
+  padding-top: 29px;
+  padding-bottom: 30px;
+}
+
+.camera {
+  width: 200px;
+  height: 200px;
+  border: 2px solid #000000;
+  box-sizing: border-box;
+}
+
+.fa-camera-retro {
```

Comment:
As above, please don't change third-party classes behaviors.
Just create own class and use it it given case.